Resources/Prototypes/Entities/Mobs/Customization/Markings/earrings.yml

```diff
@@ -0,0 +1,571 @@
+    state: drop_colored_tone_1_l
+  - sprite: Mobs/Customization/earrings.rsi
+    state: drop_colored_tone_2_l
+
+- type: marking
+  id: EarringsDropColoredRight
+  bodyPart: HeadSide
+  markingCategory: HeadSide
+  speciesRestriction: [Dwarf, Human, SlimePerson, Arachnid, Reptilian, Diona, Oni, Felinid, Vulpkanin, Harpy, Gingerbread]
+  coloring:
+    default:
+      type:
+        !type:SimpleColoring
+          color: "#9df4fa"
+  sprites:
+  - sprite: Mobs/Customization/earrings.rsi
+    state: drop_colored_tone_1_r
+  - sprite: Mobs/Customization/earrings.rsi
+    state: drop_colored_tone_2_r
+
+- type: marking
+  id: EarringsDropLongLeft
+  bodyPart: HeadSide
+  markingCategory: HeadSide
+  speciesRestriction: [Dwarf, Human, SlimePerson, Arachnid, Reptilian, Diona, Oni, Felinid, Vulpkanin, Harpy, Gingerbread]
+  coloring:
+    default:
+      type:
+        !type:SimpleColoring
+          color: "#f3d052"
+  sprites:
+  - sprite: Mobs/Customization/earrings.rsi
+    state: drop_long_tone_1_l
+  - sprite: Mobs/Customization/earrings.rsi
+    state: drop_long_tone_2_l
+
+- type: marking
+  id: EarringsDropLongRight
+  bodyPart: HeadSide
+  markingCategory: HeadSide
+  speciesRestriction: [Dwarf, Human, SlimePerson, Arachnid, Reptilian, Diona, Oni, Felinid, Vulpkanin, Harpy, Gingerbread]
+  coloring:
+    default:
+      type:
+        !type:SimpleColoring
+          color: "#f3d052"
+  sprites:
+  - sprite: Mobs/Customization/earrings.rsi
+    state: drop_long_tone_1_r
+  - sprite: Mobs/Customization/earrings.rsi
+    state: drop_long_tone_2_r
+
+- type: marking
+  id: EarringsCrescentLeft
+  bodyPart: HeadSide
+  markingCategory: HeadSide
+  speciesRestriction: [Dwarf, Human, SlimePerson, Arachnid, Reptilian, Diona, Oni, Felinid, Vulpkanin, Harpy, Gingerbread]
+  coloring:
+    default:
+      type:
+        !type:SimpleColoring
+          color: "#f4f4f4"
+  sprites:
+  - sprite: Mobs/Customization/earrings.rsi
+    state: crescent_l
+
+- type: marking
+  id: EarringsCrescentRight
+  bodyPart: HeadSide
+  markingCategory: HeadSide
+  speciesRestriction: [Dwarf, Human, SlimePerson, Arachnid, Reptilian, Diona, Oni, Felinid, Vulpkanin, Harpy, Gingerbread]
+  coloring:
+    default:
+      type:
+        !type:SimpleColoring
+          color: "#f4f4f4"
+  sprites:
+  - sprite: Mobs/Customization/earrings.rsi
+    state: crescent_r
+
+- type: marking
+  id: EarringsBangleLeft
+  bodyPart: HeadSide
+  markingCategory: HeadSide
+  speciesRestriction: [Dwarf, Human, SlimePerson, Arachnid, Reptilian, Diona, Oni, Felinid, Vulpkanin, Harpy, Gingerbread]
+  coloring:
+    default:
+      type:
+        !type:SimpleColoring
+          color: "#f3d052"
+  sprites:
+  - sprite: Mobs/Customization/earrings.rsi
+    state: bangle_l
+
+- type: marking
+  id: EarringsBangleRight
+  bodyPart: HeadSide
+  markingCategory: HeadSide
+  speciesRestriction: [Dwarf, Human, SlimePerson, Arachnid, Reptilian, Diona, Oni, Felinid, Vulpkanin, Harpy, Gingerbread]
```

Comment:
I hate that we can't just do the inverse of this by making a species Blacklist, for when you have a marking that could be put on any species. 

Also, these are missing Arachne!